account_payment_order/views/account_payment_order.xml

```diff
@@ -40,6 +40,12 @@
                         states="cancel"
                         string="Back to Draft"
                     />
+                    <button
+                        name="action_done"
+                        type="object"
+                        states="uploaded"
+                        string="Done Payments"
+                    />
```

Comment:
What's the point of the "done" state ? It was not present in my initial design of account_payment_order, and I see that it has been introduced by @JordiBForgeFlow in this commit
https://github.com/OCA/bank-payment/commit/81152bf60304b9fde3629e038056c096f956ebf5
but the commit message gives no clue about the reason for this new "done" state. For me, the final state is "uploaded"... what's the added value of the done state?
Sorry if my question is stupid...

Reply:
I'm attempting to fix this soon along the lines of what previous versions did, see https://github.com/OCA/bank-payment/pull/784#issuecomment-896785175 - when done we can add this here too in another PR

Reply:
```
2021-11-09 14:34:02,634 28 INFO KEN_Master_V14_2021_09_23 odoo.addons.base.models.ir_ui_view: action_done is not a valid action on account.payment.order

View name: account.payment.order.form
Error context:
 view: ir.ui.view(4540,)
 xmlid: account_payment_order_form
 view.model: account.payment.order
 file: /opt/odoo/kentest14-odoo/src/odoo-platform/odoo/external-src/bank-payment/account_payment_order/views/account_payment_order.xml
```
 
Getting error with latest gitaggregate